openpaas-james/jmap/extensions/src/main/scala/com/linagora/openpaas/james/jmap/json/FilterSerializer.scala

```diff
@@ -59,6 +59,7 @@ case class FilterSerializer @Inject()(mailboxIdFactory: MailboxId.Factory) {
   implicit val actionReads: Reads[Action] = Json.reads[Action]
   implicit val ruleWithIdReads: Reads[RuleWithId] = Json.reads[RuleWithId]
   implicit val updateReads: Reads[Update] = Json.valueReads[Update]
+  implicit val filterStateReads: Reads[FilterState] = Json.valueReads[FilterState]
```

Comment:
Please use format to combine the reads and the writes in a single line.

Reply:
`implicit val filterStateFormat: Format[FilterState] = Json.valueFormat[FilterState]